### ISSUE TITLE
Looks great!

### ISSUE THREAD

**Issue by arkadianriver** [FantasyLions/fantasylions.github.io#1]
It's nice to see a version with lots of posts, like yours. :O) (Don't forget to delete my posts) ;-)

**Comment by fantasylion** [comment]
ha-ha! Is realy thanks for your theme!   O(∩_∩)O~~

**Comment by arkadianriver** [comment]
:O) All thanks to aj (n33co) and his spectral theme, really. I simply jekllified it and it still needs some work (I don't like how the "pages" links on the landing page aren't fully 'clickable' and the text in the features links on the landing page don't line up right when displayed on mobile). But seriously.. the 28 hour days post.. mind if you kill it, make me the author, or merge in my latest changes from upstream? (I added a feature to hopefully have author and disqus comments travel with articles if they're forked and not deleted.) But yeah, if I could read Chinese I'd probably learn lots from your site. Looks like you've done lots of work.
